### ISSUE TITLE
Unable to print solution files (learnyounode)

### ISSUE THREAD

**Issue by TheGuardianWolf** [nodeschool/discussions#1600]
Repeatedly getting this error for some reason, tried both admin and non admin command prompts:

```
C:\Users\lichk\OneDrive\Projects\Node\Tutorial>learnyounode verify program.js

Your submission results compared to the expected:

                 ACTUAL                                 EXPECTED                
────────────────────────────────────────────────────────────────────────────────

   "CHANGELOG.md"                      ==    "CHANGELOG.md"                     
   "LICENCE.md"                        ==    "LICENCE.md"                       
   "README.md"                         ==    "README.md"                        
   ""                                  ==    ""                                 

────────────────────────────────────────────────────────────────────────────────

ERROR: There was a problem printing the solution files: testing
```

**Comment by dasDaniel** [comment]
getting the same thing on c9.io

**Comment by redragonx** [comment]
Try the fix in issue #395

Try using this in your exported function:

        if (err)
            return callback(err)

If you don't just return, the function will keep running and you don't want that.
